avocado/core/nrunner.py

```diff
@@ -54,14 +58,18 @@ class ProcessSpawner(BaseSpawner):
 
     METHODS = [SpawnMethod.STANDALONE_EXECUTABLE]
 
+    @staticmethod
+    def is_alive(task):
+        return task.spawn_handle.returncode is None
```

Comment:
Looks like negative values are also valid for terminated processes. Not sure here.

> A negative value -N indicates that the child was terminated by signal N (POSIX only).

Reply:
Well, it doesn't really matter here because not having *any* return code (positive, 0 or negative) means that the process has not returned, so, it's alive.

Reply:
Yes, you are right.